src/docs/swagger.yaml

```diff
@@ -1618,7 +1646,9 @@ paths:
     post:
       consumes:
       - application/json
-      description: 'Send a signal message. Set the text_mode to ''styled'' in case you want to add formatting to your text message. Styling Options: *italic text*, **bold text**, ~strikethrough text~.'
+      description: 'Send a signal message. Set the text_mode to ''styled'' in case
+        you want to add formatting to your text message. Styling Options: *italic
+        text*, **bold text**, ~strikethrough text~.'
```

Comment:
@bbernhard I"m not sure why most of the changes in these docs were made. I saw your comment https://github.com/bbernhard/signal-cli-rest-api/pull/209#issuecomment-1036429496 that you use `swag` to generate the docs, so I installed the swag version v1.6.7 that is pinned in the Dockerfile and ran `swag init` and these are the automated changes. I also double-checked and I am on `go` version `1.22.3` so it's not clear to me why these linting changes were made.